source/commands/env/export/components/ExportStatus.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Text } from 'ink';
+import Spinner from 'ink-spinner';
+import { ExportState } from '../types.js';
+
+interface ExportStatusProps {
+	state: ExportState;
+	file?: string;
+}
+
+export const ExportStatus: React.FC<ExportStatusProps> = ({ state, file }) => {
+	if (state.error) {
```

Comment:
Please replace the `if (state.error) {` convention with `{state.error &&` convention, to make it more readable